qa/L0_model_config/test.sh

```diff
@@ -297,7 +302,14 @@ done
 
 # Run special test cases
 for TARGET in `ls special_cases`; do
-    SERVER_ARGS="--model-repository=`pwd`/models --strict-model-config=true"
+    case $TARGET in
+        "invalid_platform")
```

Comment:
should this be `noautofill_noconfig` ?

Reply:
It is little counter-intuitive. noautofill in this model means that the underlying model doesn't support auto-complete. But to test this special scenario we need the server to run in default mode(with auto-complete enabled).

The scenario being tested here is that for models not supporting auto-complete we throw error if there is no config.pbtxt file provided.

I am open to suggestion for better nomenclature.